catalogue/webapp/components/IIIFViewer/IIIFViewer.tsx

```diff
@@ -58,6 +62,10 @@ type IIIFViewerProps = {
   transformedManifest?: TransformedManifest;
   canvasOcr?: string;
   handleImageError?: () => void;
+  resultsState: {
+    searchResults: SearchResults | undefined;
```

Comment:
np but had a comment by Alex recently on a PR of mine where they were saying we'd moved towards using optional props instead of making `undefined` an option?

Reply:
There were a few more of these in the PR, but thought I'd only ask the one time 😄